<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicInterface</class>
 <widget class="QMainWindow" name="GraphicInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>TextSearcher</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="startUrl">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>30</y>
      <width>921</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="urlLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>90</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Starting URL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Text to search:</string>
    </property>
   </widget>
   <widget class="QLabel" name="threadsNumberLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>95</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of threads:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxUrlsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>125</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Max numbers of URLs: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>60</y>
      <width>921</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="threadsNumber">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>90</y>
      <width>921</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="urlsNumber">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>120</y>
      <width>921</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>200</y>
      <width>1280</width>
      <height>580</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>580</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>580</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScrollMargin">
     <number>11</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>80</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>367</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>25</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>25</number>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">URL</string>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Search result</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="Resultlabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>155</y>
      <width>761</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
